### ISSUE TITLE
Project paths with spaces cause problems

### ISSUE THREAD

**Issue by joey-coleman** [overturetool/overture#119]
> The following bug was [originally reported](https://sourceforge.net/p/overture/bugs/119/) on Sourceforge by [nick_battle](https://sourceforge.net/u/nick_battle/profile/), 2011-11-22 21:11:28:

To quote an email bug report (symptom is a Java ClassNotFound exception when running the spec):

The problem was that the path where I had uncompressed Overture was: /home/rolando/escola/mfes/vdm/OvertureIde-1.1.0

However, /home/rolando/escola/ is a symlink that points to: /home/rolando/Área\ de\ Trabalho/feup/4_ano/

So although the `pwd` would read: /home/rolando/escola/mfes/vdm/OvertureIde-1.1.0 it was actually pointing 
to /home/rolando/Área\ de\ Trabalho/feup/4_ano/mfes/vdm/OvertureIde-1.1.0

Basically putting the vdm/ folder in my $HOME fixed the problem.

To confirm that the problem was the whitespace in the "Área\ de\ Trabalho" folder, and not, say, the character 'Á', I also uncompressed Overture to ~/Área_de_Trabalho/

It still worked fine.

So the problem seems to be that if the _real_ path to a model contains any whitespace, Overture won't be able to find the model.
